src/main/java/engine/ItemManager.java

```diff
@@ -348,6 +375,13 @@ public boolean isTimeStopActive() {
         return !this.timeStop_cooldown.checkFinished();
     }
 
+    /**
+     * Checks if AllyShip is active.
+     *
+     * @return True when AllyShip is active.
+     */
+    public boolean isAllyShipActive() { return !this.allyShip_cooldown.checkFinished(); }
```

Comment:
Please follow the code format
```suggestion
    public boolean isAllyShipActive() { 
        return !this.allyShip_cooldown.checkFinished(); 
    }
```